controls/SV-204416.rb

```diff
@@ -16,17 +15,18 @@
     Add or update the following line in "/etc/login.defs":
     ENCRYPT_METHOD SHA512'
   impact 0.5
-  tag 'legacy': ['V-71921', 'SV-86545']
-  tag 'severity': 'medium'
-  tag 'gtitle': 'SRG-OS-000073-GPOS-00041'
-  tag 'gid': 'V-204416'
-  tag 'rid': 'SV-204416r603261_rule'
-  tag 'stig_id': 'RHEL-07-010210'
-  tag 'fix_id': 'F-4540r88441_fix'
-  tag 'cci': ['CCI-000196']
+  tag legacy: ['V-71921', 'SV-86545']
+  tag severity: 'medium'
+  tag gtitle: 'SRG-OS-000073-GPOS-00041'
+  tag gid: 'V-204416'
+  tag rid: 'SV-204416r877397_rule'
+  tag stig_id: 'RHEL-07-010210'
+  tag fix_id: 'F-4540r88441_fix'
+  tag cci: ['CCI-000196']
   tag nist: ['IA-5 (1) (c)']
-  tag subsystems: ["login_defs","password"]
-  tag 'host', 'container'
+  tag subsystems: ['login_defs', 'password']
+  tag 'host'
+  tag 'container'
 
   describe login_defs do
```

Comment:
'login.defs ENCRYPT_METHOD is expected to cmp == "SHA512"'

maybe we can put login_defs as the subject and make it more clear what the test is expecting like

```
describe 'The "ENCRYPT_METHOD" configuration setting in /etc/login.defs should be "SHA512"' do
  subject login_defs
  blah
end
```

Reply:
I think we should more consistently provide human readable/actionable output throughout all of these controls.